src/KBarSearch.tsx

```diff
@@ -58,6 +58,7 @@ export function KBarSearch(
         options?.callbacks?.onQueryChange?.(event.target.value);
       }}
       onKeyDown={(event) => {
+        props.onKeyDown?.(event);
```

Comment:
Would you mind placing the spread [here](https://github.com/timc1/kbar/blob/7dd993fa320a254353d6fab4edddf91d0dc54062/src/KBarSearch.tsx#L67) as the first item under <input /> [here](https://github.com/timc1/kbar/blob/7dd993fa320a254353d6fab4edddf91d0dc54062/src/KBarSearch.tsx#L44)?

```
<input
  {...props}
  // ...
/>
```

This way an external `onKeyDown` won't override the one we've got

Reply:
Done.